code/game/barricade.dm

```diff
@@ -0,0 +1,980 @@
+					user.visible_message("<span class='notice'>[user] fumbles around figuring out how to assemble [src].</span>",
+					"<span class='notice'>You fumble around figuring out how to assemble [src].</span>")
+					var/fumbling_time = 10 * ( SKILL_CONSTRUCTION_METAL - user.mind.cm_skills.construction )
+					if(!do_after(user, fumbling_time, TRUE, src, BUSY_ICON_UNSKILLED))
+						return
+
+				for(var/obj/structure/barricade/B in loc)
+					if(B != src && B.dir == dir)
+						to_chat(user, "<span class='warning'>There's already a barricade here.</span>")
+						return
+
+				playsound(loc, 'sound/items/ratchet.ogg', 25, 1)
+				if(!do_after(user, 10, TRUE, src, BUSY_ICON_BUILD))
+					return
+
+				user.visible_message("<span class='notice'>[user] secures [src]'s anchor bolts.</span>",
+				"<span class='notice'>You secure [src]'s anchor bolts.</span>")
+				build_state = 1
+				anchored = TRUE
+				update_icon() //unanchored changes layer
+
+			else if(iscrowbar(I))
+				if(user.action_busy)
+					return
+
+				if(user.mind?.cm_skills && user.mind.cm_skills.construction < SKILL_CONSTRUCTION_METAL)
+					user.visible_message("<span class='notice'>[user] fumbles around figuring out how to disassemble [src].</span>",
+					"<span class='notice'>You fumble around figuring out how to disassemble [src].</span>")
+					var/fumbling_time = 50 * ( SKILL_CONSTRUCTION_METAL - user.mind.cm_skills.construction )
+					if(!do_after(user, fumbling_time, TRUE, src, BUSY_ICON_UNSKILLED))
+						return
+
+				user.visible_message("<span class='notice'>[user] starts unseating [src]'s panels.</span>",
+				"<span class='notice'>You start unseating [src]'s panels.</span>")
+
+				playsound(loc, 'sound/items/crowbar.ogg', 25, 1)
+				if(!do_after(user, 50, TRUE, src, BUSY_ICON_BUILD))
+					return
+
+				user.visible_message("<span class='notice'>[user] takes [src]'s panels apart.</span>",
+				"<span class='notice'>You take [src]'s panels apart.</span>")
+				playsound(loc, 'sound/items/deconstruct.ogg', 25, 1)
+				destroy_structure(TRUE) //Note : Handles deconstruction too !
+
+
+/obj/structure/barricade/metal/ex_act(severity)
+	switch(severity)
+		if(1)
+			obj_integrity -= rand(400, 600)
+		if(2)
+			obj_integrity -= rand(150, 350)
+		if(3)
+			obj_integrity -= rand(50, 100)
+
+	update_health()
+
+/obj/structure/barricade/metal/bullet_act(obj/item/projectile/P)
+	bullet_ping(P)
+	obj_integrity -= round(P.damage/10)
+
+	if(istype(P.ammo, /datum/ammo/xeno/boiler_gas))
+		obj_integrity -= 50
+
+	update_health()
+	return TRUE
+
+/*----------------------*/
+// PLASTEEL
+/*----------------------*/
+
+/obj/structure/barricade/plasteel
+	name = "plasteel barricade"
+	desc = "A very sturdy barricade made out of plasteel panels, the pinnacle of strongpoints. Use a blowtorch to repair. Can be flipped down to create a path."
+	icon_state = "plasteel_closed_0"
+	max_integrity = 600
+	crusher_resistant = TRUE
```

Comment:
We don't have crushers, so best remove this and anything it refers to

Reply:
I think I have that now in change commit 746e6d4